app/views/tracks/build/show.html.haml

```diff
@@ -200,7 +201,6 @@
                 = exercise.title
               .record-value #{exercise.num_completed} completions (30% of attempts)
 
-        = render ReactComponents::Track::Credits.new(Array.new(5, 'https://avatars.githubusercontent.com/u/66035744?v=4'), 1, "volunteer", 3, "hero")
```

Comment:
Shouldn't this be replaced with a call to the renamed component?

Reply:
I've already replaced those. check above.

Reply:
Ah